### PR TITLE
Docs/additional context

### PR DESCRIPTION
**Pull request by chopfitzroy**
Just some really basic updates to the README that caught me out when I was setting up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,8 +72,12 @@ app.on("token", async ({ token, octokit }) => {
 });
 
 require("http").createServer(getNodeMiddleware(app)).listen(3000);
-// can now receive user authorization callbacks at /api/github/oauth/callback
+// or `expressApp.use(getNodeMiddleware(app));` for Express apps
 ```
+## Endpoints
+
+- Redirect to authorization prompt at `/api/github/oauth/login`
+- Receive user authorization callbacks at `/api/github/oauth/callback`
```
Comment (gr2m) [inline README.md:80]: We document the endpoints at https://github.com/octokit/oauth-app.js#middlewares, maybe could just link to them? How about this?

```suggestion
// can now receive user authorization callbacks at /api/github/oauth/callback
// See all endpoints at https://github.com/octokit/oauth-app.js#middlewares
```

Comment (chopfitzroy) [reply README.md:80]: That is a really good idea, it just took me sometime to fund the `/api/github/oauth/login` endpoint and  wasn't sure where to look.

**Review by gr2m — APPROVED** [review]
Thanks!

File: README.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ app.on("token", async ({ token, octokit }) => {
 
 require("http").createServer(getNodeMiddleware(app)).listen(3000);
 // can now receive user authorization callbacks at /api/github/oauth/callback
-```
+// See all endpoints at https://github.com/octokit/oauth-app.js#middlewares
```
Comment (gr2m) [inline README.md:76]:
````suggestion
// See all endpoints at https://github.com/octokit/oauth-app.js#middlewares
```
````

Comment (gr2m) [reply README.md:76]: you accidentally removed the 3 backticks which end the code block syntax highlighting

Comment (chopfitzroy) [reply README.md:76]: Oops! That is the last time I try to author through the web UI 🤦‍♂️

**Review by gr2m — APPROVED** [review]
Thanks a lot! PRs improving documentation are the best PRs 💐

**Pull request merged**